src/main/resources/db/changelog/v.0.1/db.changelog-coupons.yaml

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: 13-create-coupons-table
+      author: elizaveta_dubogryz
+      changes:
+        - createTable:
+            tableName: coupons
+            columns:
+              - column:
+                  name: id
+                  type: bigint
```

Comment:
I suggest UUID as primary key here and in other tables

Reply:
+